schemas/dll.yml

```
@@ -45,3 +45,17 @@
       type: keyword
       description: Full file path of the library.
       example: C:\Windows\System32\kernel32.dll
+
+    - name: origin_referrer_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
+      description: The URL of the webpage that linked to the dll file.
+      example: http://example.com/article1.html
```

```suggestion
      example: http://example.com/article1.html
      beta: This field is beta and subject to change.
```

Please add `beta: This field is beta and subject to change.` here and to all the other new fields that are being added.

---

Thank you for the feedback! I've fixed as suggested.